tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "sourceMap": true,
-    "target": "es5",
+    "target": "ES2017",
```

Comment:
All the above changes are re-ordering things so that dependencies defined prior to their use.

Apparently required to make this 1 line change.